zello-sdk/src/com/zello/sdk/Sdk.java

```diff
@@ -954,6 +987,18 @@ static Error intToError(int error) {
 		}
 	}
 
+	static PermissionError intToPermissionError(int error) {
```

Comment:
(01) Does this method need to be static? Should it be private?

(02) Method could be simplified to one line:
`return error >= 0 && error < PermissionError.values().length ? PermissionError.values()[error]  : PermissionError.UNKNOWN`

(03) Could also annotate "error" param with @IntRange to match range of enum if desired